AppTemplate/AppTemplate/Renderer.swift

```diff
@@ -75,6 +75,17 @@ class Renderer: NSObject {
                         }
 
                         return button
+                    case "switch":
+                        let switch_: UISwitch = UISwitch(frame: CGRect.zero)
+                        applyFacts(view: switch_, facts: facts, tag: tag)
+                        if !handlers.isEmpty {
+                            let handlerNode = handlers[handlersIndex] as Json
+                            if let handlerOffset = handlerNode["offset"] as? Int, handlerOffset == offset, let funcs = handlerNode["funcs"] as? Json, let eventId = handlerNode["eventId"] as? UInt64 {
+                                addControlHandlers(funcs, id: eventId, view: switch_)
+                                handlersIndex += 1
+                            }
+                        }
```

Comment:
Just copied this handlers block, seems like a candidate for extract function at some point?